ios/Exponent/Supporting/Exponent.entitlements

```diff
@@ -11,17 +11,7 @@
 		<string>webcredentials:expo.io</string>
 	</array>
 	<key>com.apple.developer.icloud-container-identifiers</key>
-	<array>
-		<string>iCloud.$(CFBundleIdentifier)</string>
-	</array>
-	<key>com.apple.developer.icloud-services</key>
-	<array>
-		<string>CloudDocuments</string>
-	</array>
-	<key>com.apple.developer.ubiquity-container-identifiers</key>
-	<array>
-		<string>iCloud.$(CFBundleIdentifier)</string>
-	</array>
+	<array/>
```

Comment:
🤔

Reply:
I believed I removed iCloud features because it wasn't necessary for this use case. Will have to add that back in as well. That's just an option in the xcode settings.